glide.yaml

```diff
@@ -38,3 +38,4 @@ import:
 - package: golang.org/x/tools
   subpackages:
   - go/gcexportdata
+- package: github.com/Sirupsen/logrus
```

Comment:
I had his exact thought, it's strange but seemed heavy when were the only ones consuming this component.

I actually wish the go stdlib had a default logger interface, that'd be awesome